README.md

```diff
@@ -1,3 +1,13 @@
 # CQ-Rails
 
 [![Build Status](https://travis-ci.org/wolox-training/cq-rails.svg?branch=master)](https://travis-ci.org/wolox-training/cq-rails)
+
+
+base url: https://boiling-meadow-54161.herokuapp.com
+
+verb | url | use | params
```

Comment:
Hahaha, nice documentation. book index should be `/api/v1/books` :upside_down_face:

Reply:
No need to change it